backend/db/load.py

```diff
@@ -2,17 +2,54 @@
 from typing import List
```

Comment:
@joeflack4, can you go over these changes with me. I just want to understand them. (The stuff in the previous files in this merge seemed pretty clear.)